views/dashboard/partials/modal-dashboard-help.html.twig

```diff
@@ -9,18 +9,19 @@
         <ul>
           <li><strong>Enrollment Status:</strong>
             <ul>
-              <li><strong>Registered:</strong> A person who created an account by entering a name and either a mobile phone number or an email address and has chosen a language preference but has not yet completed the informed consent process. A registered participant has a unique AoURP participant identifier (PID).</li>
-              <li><strong>Participant:</strong> A person who has completed the informed consent process and is enrolled in AoURP. This individual has not yet signed the HIPAA Authorization/EHR Consent (if available and where relevant, based on AoURP capacity to effectively search for and access records) or the DV EHR Intent to Share agreement.</li>
+              <li><strong>Registered:</strong> A person who created an account by entering a name and either a mobile phone number or an email address and has chosen a language preference but has not yet completed the informed consent process. A registered participant has a unique AoURP participant identifier (PID). All registered individuals will remain in the Registered status until the primary consent is signed.</li>
+              <li><strong>Participant:</strong> A person who has completed the informed consent process and is enrolled in AoURP will become an official Participant.  All Participants will have the opportunity to respond to the HIPAA Authorization/EHR Consent or the DV EHR Intent to Share agreement. If the participant agrees to the sharing EHR data, this person will move to the next status: Fully Consented. If the individual does not agree to share EHR data, the status will remain as a participant. Participants do then have the ability to complete PPI questionnaires and become eligible for PM&B, but will not be considered Core Participants.</li>
```

Comment:
Very minor, may be we can use html entity`&amp;` for `&` here and in line 81 as well?

Reply:
👍